pkg/commands/git_commands/branch_loader.go

```diff
@@ -7,19 +7,19 @@ import (
 	"strings"
 
 	"github.com/jesseduffield/generics/set"
-	"github.com/jesseduffield/go-git/v5/config"
+	ggconfig "github.com/jesseduffield/go-git/v5/config"
```

Comment:
I gave an alias to `"github.com/jesseduffield/go-git/v5/config"` intentionally.
Because the package in same project should be prioritized.
In this  case, `"github.com/jesseduffield/lazygit/pkg/config"` should be prioritized than `"github.com/jesseduffield/go-git/v5/config"`.

`ggconfig` means GoGitConfig. If you feel wired, let me know better naming please.